server/app.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+const app = express();
+const cors = require('cors')
+
+app.use(cors());
+app.use(express.json());
+
+const search = require('./data')
+
+
+app.get('/', (req, res) => {
+    res.json('It works!')
+})
+
+//Reading aspects of the code
+
+app.get('/search', (req, res) => {
+    res.json(search)
+})
+
+app.get('/search/:name', (req, res) => {
+    try {
+        let requestedSearch = req.params.name
+        let matchingSearch = search.find((item) => item.title.toLowerCase() === requestedSearch.toLowerCase());
```

Comment:
Could perhaps have made the get request look cleaner by separating the lowercase converter into a helper fct. Only minor suggestion tho - not needed